components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/claims/impl/DefaultClaimHandler.java

```diff
@@ -1,17 +1,17 @@
-/*
- * Copyright (c) 2013, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+/**
+ * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
```

Comment:
According to my understanding we don't need to update the year in the license header if only the created year is mentioned. Shall we confirm this and update accordingly.

Reply:
resolved with https://github.com/wso2/carbon-identity-framework/pull/4244/commits/9a47021d8fafc7687f29ea4cf5cc08b7ee34953d